axmol/math/Color.h

```diff
@@ -178,6 +178,18 @@ struct AX_DLL Color : public Vec4Adapter<Color>
             return Color{r, g, b, static_cast<float>(alpha) / 255.0f};
     }
 
+    template <typename T>
```

Comment:
shoud be static method like:
 ```cpp
    static Color random()
    {
            return Color(AXRANDOM_0_1(), AXRANDOM_0_1(), AXRANDOM_0_1(), AXRANDOM_0_1());
    }
```

then usage:
```cpp
auto color1 = Color::random();        // full random color
auto color2 = color1.withAlpha(0.5f);  // only rgb is random
```

Reply:
Is it usefull to have the `alpha` also random?

There is a  `Color withAlpha(T alpha) const` returning rgb and defined alpha